src/patternfly/components/Table/examples/Table.md

```diff
@@ -1536,9 +1536,13 @@ Note: To apply padding to `.pf-c-table__expandable-row`, wrap the content in `.p
 
 | Class | Applied | Outcome |
 | -- | -- | -- |
-| `.pf-c-table__tree-view-main` | `<div>` | Initiates a tree view table main container. |
-| `.pf-c-table__tree-view-content` | `<div>` | Initiates a tree view table content element. |
-| `.pf-c-table__tree-view-icon` | `<span>` | Initiates a tree view icon wrapper. |
+| `.pf-c-table__tree-view-main` | `<div>` | Initiates a tree view table main container. **Required with tree view** |
+| `.pf-c-table__tree-view-content` | `<div>` | Initiates a tree view table content element. **Required with tree view** |
```

Comment:
I might swap `-content` and `-main`? I'm not sure if we have both `-content` and `-main` in another component, but typically `-main` wraps the title and whatever may go beside it in an item (eg, menu item, tree view/dual list selector node). WDYT?